yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -237,17 +244,13 @@ export function makeVerificationKey(): VerificationKey {
 export function makePreviousKernelData(seed = 1, kernelPublicInputs?: KernelCircuitPublicInputs): PreviousKernelData {
   return new PreviousKernelData(
     kernelPublicInputs ?? makeKernelPublicInputs(seed),
-    makeProof(seed + 0x80),
+    new Proof(Buffer.alloc(16, seed + 0x80)),
     makeVerificationKey(),
     0x42,
-    range(VK_TREE_HEIGHT, 0x1000).map(fr),
+    tupleTimes(VK_TREE_HEIGHT, fr, 0x1000),
```

Comment:
It is not super clear what `tupleTimes` is from its name

Reply:
it's based off 'times', but a tuple, I'll think on it